bigbluebutton-html5/imports/ui/components/nav-bar/settings-dropdown/container.jsx

```diff
@@ -1,59 +1,59 @@
-import React, { Component } from 'react';

```

Comment:
I just changed CRLF mode to LF mode at line separator cause lint said that there was one space at the end of each line.